frontend/public/components/deployment-config.tsx

```diff
@@ -262,6 +266,33 @@ export const DeploymentConfigDetailsList = ({ dc }) => {
   );
 };
 
+export const DeploymentConfigDeprecationAlert: React.FC = () => {
+  const { t } = useTranslation();
+  return (
+    <Alert
+      isInline
+      variant="info"
+      title={t('public~DeploymentConfig is being deprecated with OpenShift 4.14')}
+    >
+      <p>
+        {t(
+          'public~Feature development of DeploymentConfigs will be deprecated in OpenShift Container Platform 4.14.',
+        )}
+      </p>
+      <p>
+        {t(
+          'public~DeploymentConfigs will continue to be supported for security and critical fixes, but you should migrate to Deployments wherever it is possible.',
+        )}
+      </p>
+      <ExternalLink
+        href={getDocumentationURL(documentationURLs.deprecatedDeploymentConfig)}
+        text={t('public~Learn more')}
```

Comment:
@avivtur could you change this to 
```suggestion
        text={t('public~Learn more about Deployments')}
```